### ISSUE TITLE
tag placement on editset

### ISSUE THREAD

**Issue by UseDotCom** [UseDotCom/lighthouse#454]
Created by **[usedotcom](https://github.com/usedotcom)**  at 16:09Z on 02/05/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

This was originally discussed in ticket #15, although it's a totally separate issue.

From Dave:

 "apparently when I introduced tags into the equation the placement of the tags form caused issues because you cannot nest one form inside another. I took the short fix and moved the tags above the images. I can implement a fix which can solve the embedded issue and I give a time estimate of around 2-4 hours. I'm not implementing this as we're trying to get as much done as possible here."

From me:

"As for tags placement - Can we just move the table row that holds the tags below the rows that hold the images? I'm looking at a live editset page, not the template for it, so I'm not sure if there are other associated issues with doing this."

- - - -

Lighthouse ticket URL: http://use-com.lighthouseapp.com/projects/103799/tickets/35

Was assigned to **[](https://github.com/)**

**Comment by UseDotCom** [comment]
By **Dave** at 05:08Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

In this afternoon's call I revised my estimate upwards to about a day. That was a good thing. I'm about 1/2 way done with this and should have it completed tomorrow.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 18:11Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

Spoke with Jonathan. As this issue only occurs on the editset.pl page, not the viewset page, only the photo set owner sees the tags above the thumbs. Jonathan wants to just move on as it is not so important on this page. I have changed the status to Hold as he doesn't want to wait on this.

**Comment by UseDotCom** [comment]
By **Dave** at 19:21Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

This may inter-relate with the rearrange not working properly...

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 19:23Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

so does that mean that it needs to be completed or that by not trying to make it work in the position below the thumbs will resolve the rearranging issue?

**Comment by UseDotCom** [comment]
By **Dave** at 21:09Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

I don't quite understand your question but what I'm saying is that I think to fix the rearrange issue, this needs to be fixed.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 21:17Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

The tag box was placed above the thumbs to resolve the nested form issue, right?  If we leave it there and don't try the changes to the form you suggested, will the issue with the drag and drop still be affected?

**Comment by UseDotCom** [comment]
By **Dave** at 21:21Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

I think, but I am not 100% certain, that moving the tags up there screwed something up with the script that handles D&D. I think, but I am not 100% certain that moving the tags back down will resolve the issue.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 21:52Z on 02/06/2013 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/35))*

- - - -

ok. well do what you need to do. D&D has to work. Placement of the tags on the editset page is flexible.

**Comment by UseDotCom** [comment]
Imported from Lighthouse  at 20:45Z on 02/05/2015

Last assigned to **Dave**
